pkg/generate/route.go

```diff
@@ -27,13 +27,13 @@ func DefaultRoute(cr *regopapi.ImageRegistry, p *parameters.Globals) (Template,
 			},
 		},
 	}
-	if cr.Spec.TLS {
-		// TLS certificates are served by the front end of the router, so they must be configured into the route,
-		// otherwise the router's default certificate will be used for TLS termination.
-		r.Spec.TLS = &routeapi.TLSConfig{
-			Termination: routeapi.TLSTerminationReencrypt,
-		}
+
+	// TLS certificates are served by the front end of the router, so they must be configured into the route,
+	// otherwise the router's default certificate will be used for TLS termination.
+	r.Spec.TLS = &routeapi.TLSConfig{
```

Comment:
So, this change makes impossible to deploy the registry that doesn't secured by TLS?

Reply:
Perhaps we should always create cluster signed certificates and remove this option.